server.go

```diff
@@ -4562,6 +4562,10 @@ func (s *server) OpenChannel(
 			req.Err <- err
 			return req.Updates, req.Err
 		}
+
+		srvrLog.Infof("Funding fee rate was not set, using defaut "+
```

Comment:
does it make sense to remove the code above and error in case no fee rate was set? it's a bit strange that we deal with default target confirmations in two different places

Reply:
totally agree, let's error out if no feerate is specified.